.env.example

```diff
@@ -0,0 +1,6 @@
+ZAP_HUD_FILES=https://zap
```

Comment:
walking through the commits rn. haha. so this is where you set the variables to be pulled in by postHTML?

Reply:
Yup! This file is just a template, for actual dev you would: 
```
cp .env.example .env
```
And then edit for your environment. `.env` is in the `.gitignore` as often times this has values you don't want committed to repo. I added some copy in the `README` about it and @melsoriano will document in the wiki too.